anecdote/metadata.csv

```diff
@@ -0,0 +1,47 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```

Comment:
It looks like the metrics are missing descriptions. Can you provide descriptions of the metrics?

Reply:
Descriptions added